### ISSUE TITLE
mention DYLD_LIBRARY_PATH in install instructions?

### ISSUE THREAD

**Issue by whoburg** [convexengineering/gpkit#1104]
This issue is triggered by my recent experience changing from mosek 7 to mosek 8 where it took me awhile debugging to realize that I needed to edit my `.bash_profile` where I have the instruction

```
export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:/Users/whoburg/mosek/8/tools/platform/osx64x86/bin
```

I realized that we don't tell our users they need to add this to their `.bash_profile` (I think it's buried somewhere in the mosek install instructions). Is it the right way, or is my system install outdated?

@edfan might be interested in this too

**Comment by edfan** [comment]
The [linked instruction page](http://docs.mosek.com/7.0/toolsinstall/Mac_OS_X_installation.html) contains:

> In recent OS X versions there is no reliable way for normal users to change the library search path. Instead we recommend that applications are linked such that they look for the MOSEK library in a path relative to the application binary rather than in the normal library search path. This linking procedure is described in the C API manual.

Is this true? It seems like including a line instructing users to edit `~/.bash_profile` would be helpful, unless there's something that can be done on our end (we do instruct users to put MOSEK in `~/mosek`, so perhaps a search path could be hardcoded). Also, I skipped over that entire paragraph while I was installing MOSEK- seems quite easy to do.

**Comment by bqpd** [comment]
We used to automatically add something to .bash_profile, but was the docs said it's better to link the applications, so we did that; it was seemingly working until recently.

**Comment by bqpd** [comment]
The bug here wasn't fixable in .bash_profile; it was a MOSEK 8 linking bug that I've now informed them of, and temporarily fixed in #1210
